lib/cryptosuites/suites.go

```diff
@@ -191,8 +191,9 @@ var (
 		ProxyKubeClient:      RSA2048,
 		// EC2InstanceConnect has always used Ed25519 by default.
 		EC2InstanceConnect: Ed25519,
-		GitHubProxyCASSH:   Ed25519,
-		GitClient:          Ed25519,
+		// Use ECDSAP256 for KMS compatibility while avoiding RSA2048.
+		GitHubProxyCASSH: ECDSAP256,
```

Comment:
nit: for my sanity could you move this up a few lines to group it with all the other CAs? same in the other suite definitions

and while you're here we could delete all the "It is not completely implemented yet." comments, if you don't mind